pkgs/development/coq-modules/ITree/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, mkCoqDerivation, coq, version ? null , paco, coq-ext-lib }:
+
+with lib; mkCoqDerivation rec {
+  pname = "InteractionTrees";
```

Comment:
```suggestion
  pname = "ITree";
```

Reply:
ok

Reply:
Sorry. This was a wrong suggestion: this `pname` is used for computing the URL from which the sources are downloaded…

Reply:
@vbgl Yes, it is the name of the git repo.
I see that the pull request has been merged.  
Should I prepare a new patch to change it back to "InteractionTrees" ?

Reply:
Yes please.

Reply:
Done https://github.com/NixOS/nixpkgs/pull/119418